frontend/services/httpGeneral.service.js

```diff
@@ -0,0 +1,58 @@
+(function(){
+
+	'use strict';
+
+	angular
+		.module('base')
+		.service('httpGeneral',httpGeneral);
+
+	httpGeneral.$inject = [
+		'$http',
+	]
+
+	function httpGeneral($http){
+		this.httpSend = function(object){
+			if (typeof object.url !== 'string' || object.url === undefined || object.url.length === 0){
+				console.log("HTTP REQUEST EROR: REQUEST ULR IS ABSENT");
+				return;
+			}
+			if (typeof object.url !== 'string' || object.type === undefined || object.type.length === 0){
+				console.log("HTTP REQUEST EROR: REQUEST TYPE IS ABSENT");
+				return;
+			}
+			switch (object.type.toLowerCase()){
+				case 'get':
+					$http.get(object.url).then(succesfullRequest,failedRequest);
+					break;
+				case 'post':
+					$http.post(object.url,object.body).then(succesfullRequest,failedRequest);
+					break;
+				case 'put':
+					$http.put(object.url,object.body).then(succesfullRequest,failedRequest);
+					break;
+				case 'delete':
+					$http.delete(object.url).then(succesfullRequest,failedRequest);
+					break;
+				}
+			function succesfullRequest(res){
+				console.log("Succesfull Request");
```

Comment:
for debug this console.log works fine, but when we start use this service our console will be overfilled with such messages